runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -19,6 +19,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
+#include "omrcomp.h"
 #include <cstring>
```

Comment:
This belongs in `.cpp` files where needed (if at all).

Reply:
Please move the include of `<cstring>` after the includes of openj9 and omr header files (it should be within the `!defined(JFRCONSTANTPOOLTYPES_HPP_)` section).